docs/reference/evercrypt/mac/index.md

```diff
@@ -51,59 +51,7 @@ Check that a hash function is supported to be used in the HMAC construction.
 
 Write the MAC of a message (`data`) by using the hash algorithm `a` with a key (`key`) into `dst`.
  
-The recommended size for the `key` depends on the used hash algorithm (see below).
+The recommended size for the `key` depends on the used hash algorithm (see `KEY_LEN` above).
 However, the key can be any length and will be hashed if it is longer and padded if it is shorter.
-The length of `dst` depends on the output length of the used hash algorithm (see below).
+The length of `dst` depends on the output length of the used hash algorithm (see `DIGEST_LEN` above).
 
---------------------------------------------------------------------------------
-
-#### BLAKE2b
-
-```{doxygenfunction} EverCrypt_HMAC_compute_blake2b
-```
-
-The key can be any length and will be hashed if it is longer and padded if it is shorter than 128 bytes.
```

Comment:
There's information in here that's not there anymore. Is this on purpose?

Reply:
That information made it's way into `KEY_LEN` and `DIGEST_LEN` :D I will improve the descriptions...